docs/tech/geometry-columns.md

```
@@ -146,23 +166,38 @@ Zoom 9+:   geom_3857
 |--------|------|-------------|-------------|
 | `geom_3857` | MultiPolygon | `geom` | Full-resolution in Web Mercator. |
-| `geom_overview_3857` | MultiPolygon | `geom_simplified_low_3857` | **Zoom 0-2**. 50km tolerance, Douglas-Peucker. |
-| `geom_simplified_low_3857` | MultiPolygon | `geom_3857` | **Zoom 3-4**. 5km tolerance. |
+| `geom_overview_3857` | MultiPolygon | `geom_simplified_low_3857` | **Zoom 0-2**. 50km tolerance, parts under 2,500km² dropped (ADR-0031). |
+| `geom_simplified_coarse_3857` | MultiPolygon | `geom_simplified_low_3857` | **Zoom 3-4**. 10km tolerance, parts under 100km² dropped (ADR-0031). |
+| `geom_simplified_low_3857` | MultiPolygon | `geom_3857` | 5km tolerance. Input to the two rungs above; no longer read by a tile function. |
 | `geom_simplified_medium_3857` | MultiPolygon | `geom_3857` | **Zoom 5-8**. 1km tolerance. |
 
 ### Render geometry selection
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)